brainfm/__init__.py

```diff
@@ -221,7 +221,21 @@ def operation(self, **kwargs):
     },
     # getMainStations  # not implemented for now, use get_stations
     # getMainStations_NoPlans  # not implemented for now, use get_stations
-    # getStationsByID  # unknown; not seen
+    "get_stations_by_id": {
+        "name": "getStationsByID",
+        "method": "post",
+        "parameters": [
+            {
+                "name": "pid",
+                "alias": "parent_id",
+                "required": True,
+                "type": int
+            }
+        ],
+        "response": jmespath.compile(
+            "{stations: stations}"),
```

Comment:
Yes, it  seams to be the same format as from get_stations, but it is a list of stations. 
I am not used to this way of generate the return data, so I am struggling with the transform. It would be great if you would give it a try.